docs/status.md

```diff
@@ -60,7 +60,7 @@ one of the following tracking labels.
 | cbrt                     | no            | yes*         | yes*           | yes             | no          |
 | ceil                     | yes           | yes          | yes            | yes             | yes         |
 | cholesky                 | yes           | yes          | yes            | yes             | no          |
-| clamp                    | no            | yes*         | yes*           | yes             | no          |
+| clamp                    | yes           | yes          | yes            | yes             | no          |
```

Comment:
Verification should be "revisit" because [kClamp supports mixed precision](https://github.com/tensorflow/tensorflow/blob/20c6943d3cd7e07da162f7778a0af5d3776274b4/tensorflow/compiler/xla/service/shape_inference.cc#L3301-L3306). We'll need to sort this out as part of #369.

Reply:
Sure